src/backend/columnar/sql/udfs/columnar_ensure_am_depends_catalog/10.2-4.sql

```diff
@@ -5,6 +5,24 @@ CREATE OR REPLACE FUNCTION citus_internal.columnar_ensure_am_depends_catalog()
 AS $func$
 BEGIN
   INSERT INTO pg_depend
+  WITH columnar_schema_members(relid) AS (
```

Comment:
If this is achievable by introducing a new version (11.1-1.sql), then that would be preferable.

See also https://github.com/citusdata/citus/pull/5935#discussion_r875606274

Reply:
Done.